### DISCUSSION TITLE
PG and AGE math functions ambiguity

### DISCUSSION THREAD

**Discussion by Hunaid2000** [apache/age#399] [Q&A] [answered]
Hi everyone.
I noticed that most of math functions are already defined by Postgres (https://www.postgresql.org/docs/8.3/functions-math.html). So my question is why are these functions defined again by AGE?
E.g.  `atan2` in PG and `age_atan2` in AGE.

**Comment by jrgemignani** [comment, marked as answer]
A lot of PG functions can't be used directly due to differences between the Cypher specification and PG. This means that we either make our own functions or wrap PG functions or both. 

In order to eliminate the ambiguities (due to naming and compilation) between Cypher and PG names, we distinguish our functions with a prefix of **age_**

That is why there are two. If you look at the code, you'll see this as well.
